core/src/main/java/org/apache/calcite/rel/core/Project.java

```diff
@@ -70,17 +71,21 @@
 
   protected final ImmutableList<RelHint> hints;
 
+  protected final ImmutableSet<CorrelationId> variablesSet;
+
   //~ Constructors -----------------------------------------------------------
 
   /**
    * Creates a Project.
    *
-   * @param cluster  Cluster that this relational expression belongs to
-   * @param traits   Traits of this relational expression
-   * @param hints    Hints of this relation expression
-   * @param input    Input relational expression
-   * @param projects List of expressions for the input columns
-   * @param rowType  Output row type
+   * @param cluster      Cluster that this relational expression belongs to
+   * @param traits       Traits of this relational expression
+   * @param hints        Hints of this relation expression
+   * @param input        Input relational expression
+   * @param projects     List of expressions for the input columns
+   * @param rowType      Output row type
```

Comment:
Frankly speaking, I think we should avoid reflowing the comments. There's no point in changing all the previous lines when you add a new one.
See https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment

Reply:
reverted this change